--- xbmc-17.1-Krypton/tools/Linux/kodi.sh.in.orig	2017-03-31 16:19:01.557278927 +0200
+++ xbmc-17.1-Krypton/tools/Linux/kodi.sh.in	2017-03-31 16:20:26.688901651 +0200
@@ -36,19 +36,6 @@
 export MALLOC_MMAP_THRESHOLD_=131072
 
 
-# Check for some options used by this script
-while [ "$#" -gt "0" ]
-do
-    case "$1" in
-        --setlibdir)
-            LIBDIR="$2"
-            shift; shift
-            ;;
-        *)
-            shift
-            ;;
-    esac
-done
 
 migrate_home()
 {
@@ -163,7 +150,7 @@
 while [ $(( $LOOP )) = "1" ]
 do
   LOOP=0
-  "$LIBDIR/${bin_name}/${bin_name}.bin" $SAVED_ARGS
+  "$LIBDIR/${bin_name}/${bin_name}.bin" "$@"
   RET=$?
   if [ $(( $RET == 65 )) = "1" ]
   then # User requested to restart app
