--- project/cmake/addons/CMakeLists.txt.orig	2015-05-16 21:14:41.654680016 +0300
+++ project/cmake/addons/CMakeLists.txt	2015-05-16 21:16:13.099495848 +0300
@@ -265,16 +265,16 @@
           # add "kodi-platform" as a dependency to every addon
           add_dependencies(${id} kodi-platform)
 
-          set(${id}_DEPENDS_DIR ${SOURCE_DIR}/depends)
+          #set(${id}_DEPENDS_DIR ${SOURCE_DIR}/depends)
 
-          if(EXISTS ${${id}_DEPENDS_DIR})
-            include(${APP_ROOT}/project/cmake/scripts/common/handle-depends.cmake)
-            add_addon_depends(${id} ${${id}_DEPENDS_DIR})
-            if(${id}_DEPS AND NOT "${${id}_DEPS}" STREQUAL "")
-              message(STATUS "${id} DEPENDENCIES: ${${id}_DEPS}")
-              add_dependencies(${id} ${${id}_DEPS})
-            endif()
-          endif()
+          #if(EXISTS ${${id}_DEPENDS_DIR})
+          #  include(${APP_ROOT}/project/cmake/scripts/common/handle-depends.cmake)
+          #  add_addon_depends(${id} ${${id}_DEPENDS_DIR})
+          #  if(${id}_DEPS AND NOT "${${id}_DEPS}" STREQUAL "")
+          #    message(STATUS "${id} DEPENDENCIES: ${${id}_DEPS}")
+          #    add_dependencies(${id} ${${id}_DEPS})
+          #  endif()
+          #endif()
         else()
           message(FATAL_ERROR "${id}: invalid or missing addon source directory at ${SOURCE_DIR}")
         endif()
