--- project/cmake/scripts/common/addon-helpers.cmake.orig	2015-06-08 22:03:02.469336590 +0300
+++ project/cmake/scripts/common/addon-helpers.cmake	2015-06-09 06:36:59.947358400 +0300
@@ -85,7 +85,7 @@
         message(FATAL_ERROR "CMAKE_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX} differs from ${APP_NAME} prefix ${${APP_NAME_UC}_PREFIX}. Please pass -DOVERRIDE_PATHS=1 to skip this check")
       else()
         if(NOT CMAKE_INSTALL_LIBDIR)
-          set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_PREFIX}/lib/${APP_NAME_LC}")
+          set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_PREFIX}/lib64/${APP_NAME_LC}")
         endif()
       endif()
     else()
