--- tools/depends/target/ffmpeg/autobuild.sh	2015-08-30 01:17:53.042558136 +0200
+++ tools/depends/target/ffmpeg/autobuild.sh.new	2015-08-30 01:09:28.000000000 +0200
@@ -110,7 +110,7 @@
   [ "$VERSION" == "$CURVER" ] && exit 0
 fi
 
-[ -f ${ARCHIVE} ] || curl -Ls --create-dirs -f -o ${ARCHIVE} ${BASE_URL}/${VERSION}.tar.gz
+
 [ $downloadonly ] && exit 0
 
 [ -d ffmpeg-${VERSION} ] && rm -rf ffmpeg-${VERSION} && rm .ffmpeg-installed >/dev/null 2>&1
