diff -up xbmc-21.1-Omega/xbmc/interfaces/swig/CMakeLists.txt.omv~ xbmc-21.1-Omega/xbmc/interfaces/swig/CMakeLists.txt
--- xbmc-21.1-Omega/xbmc/interfaces/swig/CMakeLists.txt.omv~	2024-12-31 19:13:04.789789750 +0100
+++ xbmc-21.1-Omega/xbmc/interfaces/swig/CMakeLists.txt	2024-12-31 19:13:17.891309490 +0100
@@ -22,7 +22,7 @@ function(generate_file file)
 
   add_custom_command(OUTPUT ${CPP_FILE}
                      COMMAND ${SWIG_EXECUTABLE}
-                     ARGS -w401 -c++ -o ${file}.xml -xml -I${CMAKE_SOURCE_DIR}/xbmc -xmllang python ${CMAKE_CURRENT_SOURCE_DIR}/../swig/${file}
+                     ARGS -w401 -c++ -o ${file}.xml -xml -I${CMAKE_SOURCE_DIR}/xbmc ${CMAKE_CURRENT_SOURCE_DIR}/../swig/${file}
                      COMMAND ${Java_JAVA_EXECUTABLE}
                      ARGS ${JAVA_OPEN_OPTS} -cp "${classpath}" groovy.ui.GroovyMain ${CMAKE_SOURCE_DIR}/tools/codegenerator/Generator.groovy ${file}.xml ${CMAKE_CURRENT_SOURCE_DIR}/../python/PythonSwig.cpp.template ${file}.cpp > ${devnull}
                      ${CLANG_FORMAT_COMMAND}
