Index: xbmc-20.0rc1-Nexus/xbmc/windowing/X11/GLContextEGL.h
===================================================================
--- xbmc-20.0rc1-Nexus/xbmc/windowing/X11/GLContextEGL.h
+++ xbmc-20.0rc1-Nexus/xbmc/windowing/X11/GLContextEGL.h	2022-12-10 07:50:01.563729974 +0100
@@ -13,7 +13,7 @@
 #include "threads/CriticalSection.h"
 
 #include <EGL/eglext.h>
-#include <EGL/eglextchromium.h>
+#include <EGL/eglext_angle.h>
 #include <X11/Xutil.h>
 
 class CGLContextEGL : public CGLContext
