Index: xbmc-18.0b5-Leia/cmake/modules/FindLibDvd.cmake
===================================================================
--- xbmc-18.0b5-Leia/cmake/modules/FindLibDvd.cmake
+++ xbmc-18.0b5-Leia/cmake/modules/FindLibDvd.cmake	2018-11-04 18:09:50.143278969 +0100
@@ -142,6 +142,7 @@
     set(DVDREAD_LIBRARY ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/libdvd/lib/libdvdread.a)
     ExternalProject_Add(dvdread URL ${LIBDVDREAD_URL}
                                 DOWNLOAD_NAME libdvdread-${libdvdread_VER}.tar.gz
+				URL_HASH SHA256=a30b6aa0aad0f2c505bc77948af2d5531a80b6e68112addb4c123fca24d5d3bf
                                 DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/download
                                 PREFIX ${CORE_BUILD_DIR}/libdvd
                                 CONFIGURE_COMMAND ac_cv_path_GIT= <SOURCE_DIR>/configure
