--- a/lib/platform/util/StdString.h	2014-12-02 15:34:03.000000000 +0100
+++ b/lib/platform/util/StdString.h	2015-03-03 09:34:18.606095328 +0100
@@ -1360,7 +1360,7 @@
 inline void ssasn(std::string& sDst, const int nNull)
 {
   //UNUSED(nNull);
-  ASSERT(nNull==0);
+  ASSERT(UNUSED(nNull) == 0);
   sDst.assign("");
 }
 #undef StrSizeType
@@ -1403,7 +1403,7 @@
 inline void ssasn(std::wstring& sDst, const int nNull)
 {
   //UNUSED(nNull);
-  ASSERT(nNull==0);
+  ASSERT(UNUSED(nNull) == 0);
   sDst.assign(L"");
 }
 
